shared/actions/config/index.tsx

```diff
@@ -679,16 +680,28 @@ const toggleRuntimeStats = async () => {
     logger.warn('error toggling runtime stats', err)
   }
 }
-
-const emitStartupFirstIdle = async () => {
+const emitStartupOnLoadNotInARush = async () => {
   await Saga.delay(1000)
-  return new Promise<ConfigGen.StartupFirstIdlePayload>(resolve => {
+  return new Promise<ConfigGen.LoadOnStartPayload>(resolve => {
     requestAnimationFrame(() => {
-      resolve(ConfigGen.createStartupFirstIdle())
+      resolve(ConfigGen.createLoadOnStart({phase: 'startupOrReloginButNotInARush'}))
```

Comment:
I think the `startupOrReloginButNotInARush` phase is missing a dispatch when you log in